app/services/payment_provider_customers/gocardless_service.rb

```diff
@@ -77,5 +88,38 @@ def deliver_error_webhook(gocardless_error)
         },
       )
     end
+
+    def create_billing_request(gocardless_customer_id)
+      client.billing_requests.create(
+        params: {
+          mandate_request: {
+            scheme: 'bacs',
+          },
+          links: {
+            customer: gocardless_customer_id,
+          }
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.